From: Gunnar Wrobel <p@rdus.de>
Subject: [PATCH] t/turba/HK/GW/AlternativeNameDetailView

Also display the alternative name in the detail view.

kolab/issue3859 ([turba] optimized formatName function and fallback to company attribute) 

ISSUE: https://issues.kolab.org/issue3859

STATUS: SUBMIT

Signed-off-by: Gunnar Wrobel <p@rdus.de>

--- a/contact.php	2010-03-10 06:06:29.000000000 +0100
+++ b/contact.php	2010-03-10 06:09:21.000000000 +0100
@@ -122,9 +122,16 @@
 if (!$print_view) {
     echo $tabs->render($viewName);
 }
+
+if ($contact->hasValue('name')) {
+    $name = $contact->getValue('name');
+} elseif (isset($contact->driver->alternativeName) &&
+          $contact->hasValue($contact->driver->alternativeName)) {
+    $name = $contact->getValue($contact->driver->alternativeName);
+}
 echo '<h1 class="header">' . $own_link
-    . ($contact->getValue('name')
-       ? htmlspecialchars($contact->getValue('name'))
+    . ($name
+       ? htmlspecialchars($name)
        : '<em>' . _("Blank name") . '</em>')
     . $own_icon . '</h1>';
 $view->html();
